pep-0584.rst

```diff
@@ -0,0 +1,73 @@
+PEP: 584
+Title: January 2019 steering council election
+Version: $Revision$
+Last-Modified: $Date$
+Author: Ernest W. Durbin III <ewdurbin@gmail.com>, Nathaniel J. Smith <njs@pobox.com>
+Status: Active
+Type: Informational
+Content-Type: text/x-rst
+Created: 3-Jan-2019
+
+
+Abstract
+========
+
+This document describes the schedule and other details of the January
+2019 election for the CPython steering council, as specified in
+PEP 13. This is the first steering council election.
+
+
+Returns officer
+===============
+
+In future elections, the returns officer will be appointed by the
+outgoing steering council. Since this is the first election, we have
+no outgoing steering council, and PEP 13 says that the returns officer
+is instead appointed by the PSF Executive Director, Ewa Jodlowska.
+`She appointed Ernest W. Durbin III
+<https://discuss.python.org/t/officially-appointing-the-returns-officer-for-the-steering-council-election/603>`__.
+
+
+Schedule
+========
+
+There will be a two-week nomination period, followed by a two-week
+vote.
+
+The nomination period is: January 7, 2019 through January 20, 2019
+
+The voting period is: January 21, 2019 through Febuary 3, 2019
+
+
+Candidates
+==========
+
+Candidates must be nominated by a core team member. If the candidate
+is a core team member, they may nominate themselves.
+
+Once the nomination period opens, candidates will be listed here:
+
+* *Candidate (nominated by)*
```

Comment:
```suggestion
1. *Candidate (nominated by)*
```

Might as well make this be numeric just so we know how many nominations we have.